index.html

```diff
@@ -5817,46 +5813,46 @@ <h3>Private Browsing</h3>
         <h3>Issuer Cooperation Impacts on Privacy</h3>
 
         <p>
-It cannot be overstated that [=verifiable credentials=] rely on a high degree
-of trust in [=issuers=]. The degree to which a [=holder=] might take
-advantage of possible privacy protections often depends strongly on the support
-an [=issuer=] provides for such features. In many cases, privacy protections
-which make use of zero-knowledge proofs, data minimization techniques, bearer
-credentials, abstract claims, and protections against signature-based
-correlation, require the [=issuer=] to actively support such capabilities and
-incorporate them into the [=verifiable credentials=] they issue.
+[=Verifiable credentials=] rely on a high degree of trust in [=issuers=].
+The degree to which a [=holder=] might take advantage of possible privacy
+protections often depends strongly on the support an [=issuer=] provides for
+such features. In many cases, privacy protections make use of zero-knowledge
+proofs, data minimization techniques, bearer credentials, abstract claims,
+and protections against signature-based correlation require the [=issuer=]
+to actively support such capabilities and incorporate them into the
+[=verifiable credentials=] they issue.
```

Comment:
Where does the list of what "privacy protections make use of" end, and the list of what "require the issuer to actively support such capabilities" begin? This was clear in the older version. Wording and punctuation changes merged these two lists is a way that is nonsensical. The following is a modification of the original, which hopefully is clearer.
```suggestion
such features. In many cases, privacy protections which make use of zero-knowledge
proofs, data minimization techniques, bearer credentials, abstract claims, and
protections against signature-based correlation require active support by the
[=issuer=], who must incorporate those capabilities into the [=verifiable
credentials=] they issue.
```